cloudinit/config/cc_snap.py

```diff
@@ -61,9 +61,9 @@
         snap:
             assertions:
               00: |
-              signed_assertion_blob_here
+                signed_assertion_blob_here
```

Comment:
Excellent @lucasmoura, I'm glad this caught an invalid test config Too! Proof of a job well done.